.travis.yml

```diff
@@ -5,5 +5,7 @@ node_js:
   - "4"
   - "5"
   - "6"
+env:
+  - TZ="Europe/Copenhagen"
```

Comment:
I agree. But the reason I added that was so that the documentation tests and the tests checking the failure error messages would pass, and the trouble with those tests is that `Date`s are printed out in local time. Besides printing dates out in UTC, the only other fix I can think of is to wrap those tests around a function that checks the timezone offset and allows them to fail which I think is too complicated.